packages/ripple-nuxt-tide/lib/pages/Tide.vue

```diff
@@ -128,6 +128,9 @@ export default {
     if (this.page.sidebarComponents && this.page.sidebarComponents.length > 0) {
       this.sidebarComponents = this.$tide.getDynamicComponents(this.page.sidebarComponents, this.page.sidebar)
     }
+    if (this.page.section) {
+      this.$store.commit('tideSite/setSiteSection', parseInt(this.page.section, 10))
```

Comment:
I feel this is not same as the way you talked yesterday... this will only work in Tide page, what about other static route pages, e.g search?

Reply:
I agree - but I think we should add a ticket for refactoring middleware into composable functions and handle it there